src/ui/InputSearch/InputSearch.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { forwardRef } from 'react'
```

Comment:
Remove React import from here and add a breakline

Reply:
done !